src/resource_clients/request_queue.ts

```diff
@@ -96,6 +97,138 @@ export class RequestQueueClient extends ResourceClient {
         return cast(parseDateFields(pluckData(response.data)));
     }
 
+    async batchAddRequests(
+        requests: Array<Omit<RequestQueueClientRequestSchema, 'id'>>,
+        options: RequestQueueClientAddRequestOptions = {},
+    ): Promise<RequestQueueClientBatchAddRequestsResult> {
+        ow(requests, ow.array.ofType(ow.object.partialShape({
+            id: ow.undefined,
+        })).minLength(1).maxLength(25));
+
+        ow(options, ow.object.exactShape({
+            forefront: ow.optional.boolean,
+        }));
+
+        const response = await this.httpClient.call({
+            url: this._url('requests/batch'),
+            method: 'POST',
+            data: requests,
+            params: this._params({
+                forefront: options.forefront,
+                clientKey: this.clientKey,
+            }),
+        });
+
+        return cast(parseDateFields(pluckData(response.data)));
+    }
+
+    protected async _batchAddRequestsWithRetries(
+        requests: Array<Omit<RequestQueueClientRequestSchema, 'id'>>,
+        options: RequestQueueClientBatchAddRequestOptions = {},
+    ): Promise<RequestQueueClientBatchAddRequestsResult> {
+        ow(requests, ow.array.ofType(ow.object.partialShape({
+            id: ow.undefined,
+        })).minLength(1).maxLength(25));
+
+        ow(options, ow.object.partialShape({
+            forefront: ow.optional.boolean,
+            maxRetries: ow.optional.number.greaterThan(0),
+            delayBetweenRetriesMillis: ow.optional.number.greaterThan(0),
+        }));
+
+        const {
+            maxRetries = 9,
```

Comment:
I think the semantic for `maxRetries` is different than for the `maxRetries` in the main client class, which can be confusing - here it's the max number of attempts, but there it is actually the max number of retries (the main client will always make 1 attempt + up to `maxRetries` retries). Could you unify it so it works the same?